File: sources/askit-mvc/src/main/java/com/project/askit/entity/QuestionVote.java
package com.project.askit.entity;

public class QuestionVote {

    private Integer id;

    private Integer vote;

    private Integer questionId;

    private Integer userId;

    public QuestionVote() {
    }

    public QuestionVote(Integer vote,
                        Integer questionId,
                        Integer userId) {
        this.vote = vote;
        this.questionId = questionId;
        this.userId = userId;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getVote() {
        return vote;
    }

    public void setVote(Integer vote) {
        this.vote = vote;
    }

    public Integer getQuestionId() {
        return questionId;
    }

    public void setQuestionId(Integer questionId) {
        this.questionId = questionId;
    }

    public Integer getUserId() {
        return userId;
    }

    public void setUserId(Integer userId) {
        this.userId = userId;
    }

    @Override
    public String toString() {
        return "QuestionVote{" +
                "id=" + id +
                ", vote=" + vote +
                ", questionId=" + questionId +
                ", userId=" + userId +
                '}';
    }
}

File: sources/askit_mobile/lib/entity/question_vote.dart
class QuestionVote {
  final int id;
  final int vote;
  final int questionId;
  final int userId;

  QuestionVote({this.id, this.vote, this.questionId, this.userId});

  factory QuestionVote.fromJson(Map<String, dynamic> json) =>
      QuestionVote(id: json['id'], vote: json['vote'], questionId: json['questionId'], userId: json['userId']);

  @override
  String toString() {
    return 'QuestionVote{id: $id, vote: $vote, questionId: $questionId, userId: $userId}';
  }

  Map<String, dynamic> toJson() => {
        'id': id.toString(),
        'vote': vote.toString(),
        'questionId': questionId.toString(),
        'userId': userId.toString(),
      };
}